WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -141,8 +141,14 @@ def update_all
 
   def update
     @registration = Registration.find(params[:id])
-    was_accepted = @registration.accepted?
-    if @registration.update_attributes(registration_params)
+    @competition = @registration.competition
+    if params[:from_admin_view] && @registration.updated_at.to_datetime != params[:registration][:updated_at].to_datetime
```

Comment:
Hm...

Last-write-wins is usually good enough for web apps. If you want something more ambitious, it would make more sense to do it across all operations, not a one-off thing.

Unless there is something real special about this case.